data/editor.config

```diff
@@ -1154,14 +1053,11 @@
         <include group="address" />
       </type>
       <type id="man_made-surveillance">
-      </type>
```

Comment:
Please return back this line, it breaks the XML file.
And make sure that validator runs well:
`xmllint -noout -schema data/config.xsd data/editor.config`